include/openspace/properties/numericalproperty.inl

```diff
@@ -61,6 +61,7 @@ T NumericalProperty<T>::minValue() const {
 
 template <typename T>
 void NumericalProperty<T>::setMinValue(T value) {
+    Property::notifyMetaDataChangeListeners();
     _minimumValue = std::move(value);
```

Comment:
I feel we should call the notify function _after_ setting the value if we can?  Otherwise if someone is listening to the notify change and requests the meta data, they will get the old value back

Reply:
Yes, the notification has to happen after the change!  Now, we send invalid information 

To make sure it works correctly @ylvaselling you may use the renderablenodearrow example and set the Use relative Length or Use relative Offsets property. This renderable has a function that sets the min and max when changing two properties to be "relative" or not. The min and max settings are currently not correct

https://github.com/OpenSpace/OpenSpace/blob/93ef632663009f0b3b31eff4f311560764c5fcce/modules/base/rendering/renderablenodearrow.cpp#L174